crates/sui-types/src/sui_system_state.rs

```diff
@@ -363,7 +372,15 @@ where
         .data
         .try_as_move()
         .ok_or(SuiError::SuiSystemStateNotFound)?;
-    let result = bcs::from_bytes::<SuiSystemState>(move_object.contents())
+    let result = bcs::from_bytes::<SuiSystemStateWrapper>(move_object.contents())
         .expect("Sui System State object deserialization cannot fail");
     Ok(result)
 }
+
+pub fn get_sui_system_state<S>(object_store: S) -> Result<SuiSystemState, SuiError>
+where
+    S: ObjectStore,
+{
+    let wrapper = get_sui_system_state_wrapper(object_store)?;
+    Ok(wrapper.system_state)
```

Comment:
In your next PR which will make the actual system state a dynamic field of the wrapper, I suppose you will be loading the dynamic field here? Do we have use cases where we may want to load not just the current version of the system state, but also the previous versions? If so this function should probably take in a version number too.

Reply:
I haven't seen a need for that, but will keep it in mind